src/core/filters/meta/filter.rs

```diff
@@ -456,7 +462,7 @@ impl FilterMeta {
 
                     stored_offset = offset;
                 }
-                None => bail!("{field} not found!"),
+                None => bail!("{field} not found or is a bitfield!"),
```

Comment:
Those two reasons seems a bit unrelated. Maybe `walk_btf_node` could return a `Result` instead of an `Option`, with proper explanations on why it failed to find a field?

Reply:
> Those two reasons seems a bit unrelated. Maybe `walk_btf_node` could return a `Result` instead of an `Option`, with proper explanations on why it failed to find a field?

that's what I meant with `there's no separate bail message, but it's fine as this is temporary and bitfields support is the next in line`. For that reason, it was not turned into `Result`.
This part will be removed anyway. We can change the message to "or not supported" instead of turning `Option` to `Result` otherwise I guess `Result` will be temporary but fine.

Reply:
Ah sorry, got it. That make sense.